lib/Doctrine/Migrations/DependencyFactory.php

```diff
@@ -71,6 +75,19 @@ public function __construct(Configuration $configuration, Connection $connection
         $this->em            = $em;
     }
 
+    public function freeze() : void
+    {
+        $this->frozen = true;
+        $this->configuration->freeze();
+    }
+
+    private function checkFrozenDependencies() : void
```

Comment:
Private methods usually defined after public. Does Doctrine follows same conversation? Also what do you think about start this method name with assert prefix, like assertNotFrozen?

Reply:
We don't AFAIK, we usually enforce our conventions. Love the `assertNotFrozen` suggestion 👍